sorting/wiggle_sort.cpp

```diff
@@ -0,0 +1,132 @@
+/**
+ *  \addtogroup sorting Sorting Algorithms
+ *  @{
+ *  \file
+ *  \brief [Wiggle Sort Algorithm]
+ *  (https://khanhng.com/post/wiggle-sort/) Implementation
+ *
+ *  \author [Roshan Kanwar](http://github.com/roshan0708)
+ *
+ *  \details
+ *  Wiggle Sort sorts the array into a wave like array.
+ *  An array ‘arr[0..n-1]’ is sorted in wave form,
+ *  if arr[0] >= arr[1] <= arr[2] >= arr[3] <= arr[4] >= …..
+ *
+ *  \example
+ *  arr = [1,1,5,6,1,4], after wiggle sort arr will become equal to [1,1,6,1,5,4]
+ *  arr = [2,8,9,1,7], after wiggle sort arr will become equal to [8,2,9,1,7]
+ */
+
+#include <iostream>
+
+class Algorithm {
```

Comment:
Provide a brief description of what the class does here (in the code).
```suggestion
/**
 * @brief description
 */
class Algorithm {
```